tests/server/config_test.py

```diff
@@ -0,0 +1,63 @@
+"""
+Test script for config
+"""
+
+import logging
+logger = logging.getLogger('config_test')
+
+import os
+import sys
+import time
+import random
+import shutil
+import tempfile
+import unittest
+
+from rest_tools.server import from_environment
+
+
+class config_test(unittest.TestCase):
+    def setUp(self):
+        super(config_test,self).setUp()
+        self.test_dir = tempfile.mkdtemp(dir=os.getcwd())
+        def cleanup():
+            shutil.rmtree(self.test_dir)
+        self.addCleanup(cleanup)
+        environ = os.environ.copy()
+        def clean_env():
+            for k in list(os.environ):
+                if k not in environ:
+                    del os.environ[k]
+        self.addCleanup(clean_env)
+
+    def test_01_from_environment(self):
+        with self.assertRaises(Exception):
+            from_environment({'FOO': None})
+
+        os.environ['FOO'] = 'bar'
+        config = from_environment({'FOO': None})
+        self.assertEqual(config['FOO'], 'bar')
+
+    def test_10_from_environment(self):
+        os.environ['FOO'] = 'bar'
+        with self.assertRaises(Exception):
```

Comment:
And this would be a `TypeError`? It would be beneficial to be able to narrow down what lines these are intending to fail on.